### DISCUSSION TITLE
why overseerr develop branch?

### DISCUSSION THREAD

**Discussion by nmbrg** [tteck/Proxmox#2295] [General]
what is the reason behind choosing to use the develop branch instead of stable/master?

**Comment by tteck** [comment]
It's built from source.
https://github.com/tteck/Proxmox/blob/781d9a0b2aa98c6d49253e56d269e81fa4b15983/install/overseerr-install.sh#L41-L44

**Comment by tteck** [comment]
Are you having issues?

**Comment by nmbrg** [comment]
ok, weird that they have the develop branch instead of stable beta. was pretty sure they changed it like a year ago to recommend stable beta instead of dev branch.

nah, no issues now, for some reason (guessing its the dev branch that works better than stable beta)

if i run the stable beta overseerr is crippling slow. and tried everything, and i mean EVERYTHING (complete new installation, ISP change(!), HW change, DNS.. you name it..) and the only thing that made it usable again was the dev branch version. (tho they seem to be the same version numbers)

so at the moment im glad its the dev branch beeing used. but im not that fond of getting every little tiny bit of change (most of which breaks stuff) on the dev branch.
